Open Judge System/OJS.Common/GlobalConstants.cs

```diff
@@ -118,6 +118,9 @@ public static class GlobalConstants
         public const int TestInputMinLength = 1;
         public const int TestOutputMinLength = 1;
 
+        // JVM Startup Memory
```

Comment:
Discussed with Yana, because it is used to mark the group, not to explain the variable.